File: dungeons-1.21.11/src/main/java/fishmod/utils/rendering/RenderingEvent.java
package fishmod.utils.rendering;

import net.fabricmc.fabric.api.client.rendering.v1.world.WorldRenderContext;
import net.minecraft.client.render.VertexConsumer;
import net.minecraft.client.util.math.MatrixStack;

public interface RenderingEvent {
    void render(WorldRenderContext context, MatrixStack matrixStack, VertexConsumer consumer);

}

File: dungeons-1.21.11/src/main/kotlin/fishmod/utils/rendering/RenderingEvent.kt
package fishmod.utils.rendering

import net.fabricmc.fabric.api.client.rendering.v1.world.WorldRenderContext
import net.minecraft.client.render.VertexConsumer
import net.minecraft.client.util.math.MatrixStack

fun interface RenderingEvent {
    fun render(context: WorldRenderContext, matrixStack: MatrixStack, consumer: VertexConsumer)
}
